src/actions/app.cr

```diff
@@ -183,16 +174,17 @@ module DppmRestApi::Actions::App
   end
   # Stream the logs for the given application over the websocket connection.
   relative_ws "/:app_name/logs" do |sock, context|
-    app_name = context.params.url["app_name"]
     if context.current_user? && Config.has_access? context, Access::Read
       # TODO: stream logs to sock
+      sock.tap do
+        # trick ameba into ignoring the sock variable
+      end
```

Comment:
... does renaming `sock` into `_`  can work, too?

Reply:
it does. The reason I didn't do that was I wanted a note of what it was for later, but I can just as easily add a comment.

Reply:
Having a comment sounds better than having a dummy code specifically for ameba.

Reply:
I was actually kind of hesitant to add Ameba to the CI because of situations like this, but I discovered a few bugs in the process of adding it so it seems worth it.